src/oss-tests/ProjectManagerTests/ProjectManagerFactoryTests.cs

```diff
@@ -92,10 +93,10 @@ public void AddTestManagerSucceeds()
     [TestMethod]
     public void OverrideManagerSucceeds()
     {
-        _managerOverrides[NuGetProjectManager.Type] =
-            (destinationDirectory, timeout) => new NuGetProjectManager("nugetTestDirectory", null, _httpClientFactory); // Override the default entry for nuget, and override the destinationDirectory.
+        _managerOverrides[BaseNuGetProjectManager.Type] =
+            (destinationDirectory, timeout, packageUrl) => new NuGetProjectManager("nugetTestDirectory", null, _httpClientFactory); // Override the default entry for nuget, and override the destinationDirectory.
         _managerOverrides[NPMProjectManager.Type] =
-            (destinationDirectory, timeout) => new NPMProjectManager("npmTestDirectory", null, _httpClientFactory); // Override the default entry for npm, and override the destinationDirectory.
+            (destinationDirectory, timeout, packageUrl) => new NPMProjectManager("npmTestDirectory", null, _httpClientFactory); // Override the default entry for npm, and override the destinationDirectory.
```

Comment:
I didn't see an override / more than two package managers for NPM. What is this testing here?